### PR TITLE
SOC-2299 | posts title colors fixed for The Dark Side

### PR DESCRIPTION
**Pull request by rafalkalinski**
## Links

https://wikia-inc.atlassian.net/browse/SOC-2299

## Description

On the main post list, the title is the same color as the body text, but in bold, so it's readable.
However, on the post details page, the title is in a gray color and very hard to see.

## Reviewers

@kvas-damian, @bkoval

### CODE REVIEW

**Comment by bkoval** [comment]
LGTM $$$

**Comment by kvas-damian** [comment]
LGTM :shipit:

File: front/main/app/styles/module/_discussion.scss [hunk 1]
```diff
@@ -190,8 +190,13 @@ body.discussions {
 			display: block;
 		}
 		.post-title {
+			color: $discussion-post-title-color;
 			line-height: $line-height-smallest;
 			margin: 0;
+
+			.post-details-link {
```
Comment (hakubo) [inline front/main/app/styles/module/_discussion.scss]: 4 levels deep nesting - do we have to go that deep?

Comment (rafalkalinski) [inline front/main/app/styles/module/_discussion.scss]: improved

**Pull request merged**